.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -750,4 +781,588 @@ phases:
         inputs:
           artifactName: 'publishing-info-1804-gen2-gpu'
           targetPath: 'vhd-publishing-info.json'
+        condition: and(eq(variables.DRY_RUN, 'False'), ne(variables['MODE'], 'sigMode'))
+  - phase: build_vhd_1804_containerd
```

Comment:
I know you didn't introduce this, but it's really not easy to maintain all the 9 VHD script separately while most of them are the same. Have you tried to make it reusable?

Reply:
very good point. i had a lot of trouble setting this up initially as well b/c of all the copy+pasting. I'll spend some time making this more compact.